recipes/qupy/meta.yaml

```
@@ -0,0 +1,40 @@
+{% set name = "qupy" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/ken-nakanishi/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 78363551d54e94f7cc1cd84f52bfb79442bcc1e03a918f352cf14b86fec86d5d
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - numpy
```

Please pin compatible numpy
```suggestion
    - {{ pin_compatible('numpy') }}
```

---

Thanks!